CONTRIBUTING.md

```diff
@@ -26,110 +47,274 @@ Before we get started, here are a few things we expect from you (and that you sh
+|`help wanted`|Extra attention is required|
+|`requires research`|Further information is required|
+|`priority`|High priority issue|
+|`design needed`|Requires a wireframe|
+|`tests`|For unit and integration tests|
+|`0`|Insignificant task (minimal effort and time needed)|
+|`1`|Simple issues requiring minimal effort|
+|`2`|Not so complex, requiring a relatively good amount of effort and time to fix|
+|`5`|Complex task|
+|`8`|Very complex task|
+
+> **Story point estimation**<br> 
+> Story points are used to estimate the effort needed for each task. We try to avoid using time estimates and a more useful measurement for team capacity is the effort needed and then complexity of a task.
+
+| Story Point | Complexity reference  |
+|--|--|
+| 1 | Changing a label |
+| 2 | Making a model change |
+| 3 | Implementing CRUD on the UI |
+| 5 | Implementing a new model|
+| 8 | Introducing complex logic|
+| 13| Research task, building a proof of concept |
+| 21| Should be used sparingly|
+
+## Milestones
+
+We work toward a **Sprint** with the goal to release to `production` every two weeks. We use **Github Milestones** to track what issues contributors work on during a specific Sprint. Each milestone will have a start and end date and the product team will help to groom the issues and tag them to a specific sprint number before the sprint starts.
+
+Each week we have two calls as a team to discuss our priorities and reflect:
+* **Sprint Planning:** every Monday 7 PM EAT
+* **Sprint Review:** every Saturday 7 PM EAT
+
+> Join calls using `Google Meet` and follow the `#sprint-planning channel` on slack.
+
+## Task Board
+We use **Github Projects** to manage our current sprints with automated workflows. Each repo will have its own task board to manage issues and will consist of the following columns:
+- `To do`
+- `In progress`
+- `Pull request`
+- `Dev testing`
+- `Staging`
+- `Done`
+
+Before every sprint, we archived all issues in the `Done` columns and add issue cards with the current sprint milestone tagged to them. During our sprint calls, the team will groom, discuss, and assign cards to contributors to work on.
+
+> **Note:**
+> * At the beginning of every milestone, we include all items in the `To do` stage and move them over as we go. 
+> * At the end of a milestone, any issues in `To do` or `In Progress` need to be moved over to the next sprint (if still relevant) or turned into new issues.
+
+### Triage
+
+Bugs and features will be assigned a sprint, and within a sprint, they will be assigned a priority. The priority dictates the order in which issues should be addressed. An important bug (something that we think is critical for the milestone) is to be addressed before the other bugs.
+
+### Hardening Sprint
+
+The final week of the iteration is what we call the "Hardening Sprint". During this week we will fix the critical bugs and focus the entire week on improve test coverage by building unit and integration tests for any new features. 
```

Comment:
"During this week we will fix the critical bugs and focus the entire week on improving test coverage" 
Currently it says "improve" not "improving"